### PR TITLE
[BAU-327] docs: tooltip migration guide

### PR DESCRIPTION
**Pull request by gui-santos**
# Purpose of PR

It adds the migration guide for Tooltip

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

**Review by burakukula — APPROVED** [review]

**Comment by gui-santos** [comment]
@alstaicu I merged the code before your review, I'm sorry

you can still review it and I will open a new PR with all the suggestions you may have 🙏

**Comment by alstaicu** [comment]
Hey, no worries. I'll have a look and if there's anything major, we can create a new PR. Thank you!

File: MIGRATION.md [hunk 1]
```diff
@@ -1408,3 +1410,30 @@ import {
   <Button variant="primary" size="small" />
 </Modal.Controls>;
 ```
+
+## Tooltip
+
+In version 4, we rename two props of the Tooltip: `containerElement` and `place`.
```
Comment (alstaicu) [inline MIGRATION.md:1416]: I would rephrase this as:

In version 4, we renamed two Tooltip props: `containerElement` is now `as` and `place` is now `placement`.

File: packages/forma-36-website/src/content/migration-v3-to-v4.mdx [hunk 2]
```diff
@@ -46,6 +46,8 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 - **Modal**: We changed the way you import the Modal component. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-modal-components).
 
+- **Tooltip**: Two props got renamed: `containerElement` to `as`; and `place` to `placement`. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tooltip-components).
```
Comment (alstaicu) [inline packages/forma-36-website/src/content/migration-v3-to-v4.mdx:49]: We should remove the semicolon before **and**. Other than that, it looks great.

Comment (gui-santos) [reply packages/forma-36-website/src/content/migration-v3-to-v4.mdx:49]: Cool! Thanks a lot, I will do the changes
